e2e/environment.ts

```diff
@@ -1,5 +1,5 @@
 export const environment = {
-  HOST: '192.168.1.197',
+  HOST: '10.2.5.20',
```

Comment:
this should not be needed, even we shall need a script to find and replace this when run on CI